packages/client/src/editor.tsx

```diff
@@ -6,24 +6,38 @@ import CodeMirror from "codemirror";
 import "codemirror/mode/markdown/markdown.js";
 import "codemirror/lib/codemirror.css";
 import { WebsocketProvider } from "y-websocket";
+import { useSelector } from "react-redux";
+import { selectDocument } from "./redux/document";
 
-const ROOM_NAME = "new-room";
 const SIGNALLING_SERVICE =
   process.env.STORYBOOK_SIGNAL_URL || "ws://localhost:6006/ws/signal";
 const PROVIDER_SERVICE =
   process.env.STORYBOOK_PROVIDER_URL || "ws://localhost:6006/ws/provider";
 
 export default function Editor() {
+  const doc = useSelector(selectDocument);
+  if (!doc) {
+    return <p>Select a document</p>;
+  }
+  return <TextCanvas document_id={doc.id} />;
```

Comment:
I think this is going to cause problems when the selected document changes, but we'll have to come back to that.

Reply:
I'll be on the lookout for issues relating to that